### DISCUSSION TITLE
Explore: Share shortened link button is hard to find

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#52353] [Explore]
Feedback from the user interview. When user was clicking in query history, he noticed that the create link creates shortened url. And then he mentioned that it would be nice to have this also on Explore page. Which we have, but maybe icon-button next to Explore name, makes it harder to find/understand what it does.

**Comment by radiorental** [comment]
The fundamental issue here is that it's bad UX practice to have icons without text labels.  There are very few icons that are universally understood, share is definitely not one of them

If icon + label is not an option here I'd strongly recommend at least changing the icon from [share] to [link].  The link icon is more universally understood and is more focused on the action at hand.  I.e. Share = share something?  Link = create a link

**Comment by Elfo404** [comment]
with the introduction of the new top nav, the share button is placed consistently with the dashboard share button:
![Screenshot 2023-01-11 at 14 53 40](https://user-images.githubusercontent.com/1170767/211838291-910d9822-90e4-4c34-a258-769ebb5e938f.png)
![Screenshot 2023-01-11 at 14 53 54](https://user-images.githubusercontent.com/1170767/211838293-f4a623c7-0f3d-4694-ae97-b08fa4b3acd7.png)

this may help discoverability. we are already tracking this button usage, it'd be nice to compare the usage when topnav is enabled and disabled.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
